run.py

```diff
@@ -114,6 +115,7 @@
   -c --custom-config <name>=<value> Add a custom config key/value to ceph_conf_overrides
   --custom-config-file <file>       Add custom config yaml to ceph_conf_overrides
   --xunit-results                   Create xUnit result file for test suite run [default: false]
+  --enable-eus                      Enable EUS [default: false]
```

Comment:
Would you please provide a better description here? "Enable RHEL EUS on every cluster node" or something.

Note that if a cephci user specifies `--enable-eus` and we enable the RHEL 7.7  EUS repositories, but a user also picks "rhel-7.9" VMs in OpenStack, this test is not going to be valid. I'm not sure where you would like to handle that type of input validation, if at all.

Reply:
help text added and I have added a flag in inventory file which tells if that inventory has eus support or not